src/iblt.cpp

```diff
@@ -0,0 +1,266 @@
+#include "iblt.h"
```

Comment:
missing copyright header

Reply:
I have a question here. I lifted this code from Gavin's repo. It is distributed with a MIT license. I did make some edits though. How should I copyright this while also crediting the original author?

Reply:
you should add use the original copyright notice (the one that present on Gavin repo and any other copyright that Gavin included if any), then add yours and BU ones if you wanted to.

Reply:
There is no copyright in any single file, but the top level of the repository contains the following file. So I guess I should paste the contents into each iblt file and then include the BU copyright just above it?

https://github.com/gavinandresen/IBLT_Cplusplus/blob/master/LICENSE

Reply:
Again IANAL, but this what I would do :smile:

Reply:
Ok fair enough! I pushed license information for the IBLT code.

Reply:
What do you make of wallet.py test failing due to KeyboardInterupt in CI? This just happened after I committed the license changes to the IBLT code; doesn't seem like it could be related. For me all unit tests pass in my devenv, at least when I run make check. Is there another test suite I should be running as well?

Reply:
this is a spurious failure due to travis problem. Going to rekick the failing job

Reply:
Ah I see, intermittent failure. Looks green now. Thanks.